tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -217,6 +217,18 @@ Verify Monitoring Stack Is Reconciled Without Restarting The ODS Operator
     Replace "Prometheus" With "Grafana" In Rhods-Monitor-Federation
     Wait Until Operator Reverts "Grafana" To "Prometheus" In Rhods-Monitor-Federation
 
+Verify Node Failures Are Handled
+    [Documentation]    Verifies if ODS is prepared to handle node failures
+    [Tags]    Sanity
+    ...       Tier1
```

Comment:
@pablofelix Shouldn't this fall under Tier2 or Tier3 since it is somewhat related to Fault testing ?

Reply:
Changing it to Tier2

Reply:
Done